resources/js/index.scss

```diff
@@ -20,6 +20,9 @@
     &.secondary {
       @apply p-2 rounded border border-cyan-500 text-cyan-500 disabled:bg-opacity-50;
     }
+    &.secondarySmall {
```

Comment:
może-ten-case? zamiast tenCase. Spójniej z tym jak są nazywane klasy w tailwindzie

Reply:
No, to tak jak już odpisywałem, wtedy jest mniej spójnie z tym jak nazywane są nasze klasy w *.module.scss. Mi ta niespójność tutaj nie przeszkadza za bardzo a nawet wolę widzieć że to nasze klasy.